Library/Homebrew/compat/hardware.rb

```diff
@@ -1,32 +1,54 @@
 module Hardware
   class << self
-    # We won't change the name because of backward compatibility.
+    # We won't change the names because of backward compatibility.
     # So disable rubocop here.
-    def is_32_bit? # rubocop:disable Style/PredicateName
+    # rubocop:disable Style/PredicateName
```

Comment:
This comment needs to stay on the same line like the `def` to have the desired effect. I don't think we want to disable this style check for the rest of the entire file, though maybe that's what you indeed intended …